### PR TITLE
[CI:DOCS]swagger cleanup and left-hand nav

### PR DESCRIPTION
**Pull request by baude**
add a static tags file so we can dictate the left-hand navigation. in
doing so we now override the tag in the swagger:operation. we now have
images and images (compat) as a way to differentiate.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/bindings/errors.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package bindings
+
+import (
+	"encoding/json"
+	"github.com/sirupsen/logrus"
```
Comment (mheon) [inline pkg/bindings/errors.go]: Import ordering

File: pkg/bindings/containers.go [hunk 2]
```diff
@@ -109,12 +109,14 @@ func (c Connection) UnpauseContainer(nameOrID string) error {
 }
 
 func (c Connection) WaitContainer(nameOrID string) error {
-	_, err := http.Post(c.makeEndpoint(fmt.Sprintf("containers/%s/wait", nameOrID)), "application/json", nil)
+	// TODO when returns are ironed out, we can should use the newRequest approach
+	_, err := http.Post(c.makeEndpoint(fmt.Sprintf("containers/%s/wait", nameOrID)), "application/json", nil) // nolint
```
Comment (mheon) [inline pkg/bindings/containers.go]: Shouldn't this have a closeResponseBody?

Comment (baude) [reply pkg/bindings/containers.go]: it isnt complete, hence the note

**Pull request merged**